src/cp/cp.rs

```diff
@@ -10,13 +10,32 @@
  * that was distributed with this source code.
  */
 
+extern crate libc;
 extern crate clap;
 extern crate walkdir;
+extern crate filetime;
+use filetime::FileTime;
```

Comment:
Please separate the `extern crate` and `use` statements into separate sections (with the `use` section below the `extern crate` one).